backend/rest_api_service/models.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, event_id, created_by, event_name, event_description, event_da
     def create(event: str):
         event = json.loads(event)
         return DBEvent(
-            event['event_id'],
+            None,
```

Comment:
zamiast dawać None, nie chcemy tutaj nic podawać. Dodaj do atrybutu DBEvent.event_id parametr Sequence('db_event_event_id_seq'), żeby generować automatycznie to id

https://stackoverflow.com/questions/21328599/why-isnt-sqlalchemy-creating-serial-columns